src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,14 @@
+<hibernate-configuration xmlns="http://www.hibernate.org/xsd/orm/cfg">
+    <session-factory>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/movie_app?serverTimezone=UTC</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">12345678</property>
+        <property name="dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>

```

Comment:
just to split that wall of code into logical sections
```suggestion
        <property name="hbm2ddl.auto">create-drop</property>

```

Reply:
for some reason mvn said me that I should remove this split nw when I get it back it stays quiet(